src/Uno.UI/UI/Xaml/UIElement.Skia.cs

```diff
@@ -123,7 +123,7 @@ private void OnChildAdded(UIElement child)
 		internal virtual void OnElementLoaded()
 		{
 			IsLoaded = true;
-			foreach (var innerChild in _children)
+			foreach (var innerChild in _children.ToArray())
```

Comment:
Not sure which is more performant, `ToArray` or `ToList`? I would suppose `ToList` as it remove the need to trim the array.

Reply:
It's the same underneath. (List creates an array of the right size)